process_sample_data.py

```diff
@@ -67,33 +67,41 @@ def _parseData(self):
         tidal_volume_marker = "TidalVol:"
         pressure_marker = "Pressurex10:"
```

Comment:
Nitpick, and perhaps I'm ignorant of this project's coding style guidelines, but constants are often expressed as `ALL_CAPS`, especially when there's no type system to signify their purpose.